hdfs/client.py

```diff
@@ -277,6 +278,21 @@ def status(self, hdfs_path, strict=True):
     res = self._get_file_status(hdfs_path, strict=strict)
     return res.json()['FileStatus'] if res else None
 
+  def get_acl(self, hdfs_path, strict=True):
```

Comment:
Could we rename this to `acl_status`? (To be consistent with the other methods, which don't have the `get_` prefix.)